test/spec/provider/zeebe/BusinessRuleImplementationProps.spec.js

```diff
@@ -366,20 +356,3 @@ function getTaskHeaders(element) {
 
   return getExtensionElementsList(businessObject, 'zeebe:TaskHeaders')[ 0 ];
 }
-
-async function expectEdited(container, exists) {
-
-  await wait(50);
```

Comment:
Why would the `waitFor` utility from testing-library not work here, and keeping the test?

Reply:
I don't know but it failed even after rewrite. I can try again today.

Reply:
So this is how it looks with `waitFor`, but it fails on MacOS in chromeheadless: https://github.com/bpmn-io/bpmn-js-properties-panel/pull/1111/commits/9abdfdbe04510ce67f694df4675c3864c207ee4d
It works in `.only` though.

Reply:
I checked how the test suite behaves with different test suites, and it is green when I run only `provider` tests, but starts to fail when everything is included. Perhaps it's a slow clean up that jams on a local machine?